OracleBI/README.md

```diff
@@ -175,7 +175,15 @@ In this mode, the containers are started in parallel and the BI container must w
 4. If the container is restarted, it requires the same set of environment variables passed in again, even though the domain has already been created.
 
 5. Docker health-check is not implemented.
- 
+
+6. Oracle Business Intelligence R Installer will fail as `make` is not installed.
```

Comment:
If the Dockerfile is designed to avoid this issue, do we need to keep this documented at all?

Reply:
Still wondering if this is necessary to be documented.

Reply:
The R Installer != the core product installer.  The Dockerfile needs to correctly run the core product installer, and it will not work properly if make is present without GCC.

The R Installer is an optional manual step that can be used once the container is running.  It adds additional components which cannot be packaged by Oracle due licensing.

It fails if make is not available, hence this comment.

The correct resolution would be for the R Installer to install make - it already installs a bunch of other packages.  However that bug is in the shipped product.  If PM request, I can fix the Dockerfile to install both make and GCC, but I need to cycle that through corparch.

Reply:
No need to cycle with corparch. Make and GCC are Oracle Linux packages. Just install what is needed for your image to work as best as possible OOTB.